packages/grafana-data/src/dataframe/index.ts

```diff
@@ -7,5 +7,10 @@ export * from './dimensions';
 export * from './ArrayDataFrame';
 export * from './DataFrameJSON';
 export * from './frameComparisons';
-export { anySeriesWithTimeField, isTimeSeriesFrame, isTimeSeriesFrames } from './utils';
+export {
+  anySeriesWithTimeField,
+  isTimeSeriesFrame,
+  isTimeSeriesFrames,
+  shiftComparisonFramesTimestamps,
```

Comment:
does this need to be exported yet?  can it be an internal detail that we improve before making public?